source/overview/index.rst

```diff
@@ -13,7 +13,9 @@ Below is an overview of our features, security, account management, deployment,
    /about/product
    /about/security*
    /about/integrations*
-   /about/licensing-and-subscription*
+   /about/subscription*
+   /about/cloud-subscriptions*
+   /about/self-hosted-subscriptions*
```

Comment:
```suggestion
   /about/self-hosted-subscriptions
```

Reply:
I'm reasonably sure the asterisks here were legacy holdovers that are no longer needed. We'll be able to confirm this using the latest preview.